docs/Documentation-CHN/UserGuide/9-Tools-Hive.md

```diff
@@ -0,0 +1,190 @@
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<!-- TOC -->
+## 概要
+
+- TsFile的Hive连接器使用手册
+	- 什么是TsFile的Hive连接器
+	- 系统环境要求
+	- 数据类型对应关系
+	- 为Hive添加依赖jar包
+	- 创建Tsfile-backed的Hive表
+	- 从Tsfile-backed的Hive表中查询
+	    - 选择查询语句示例
+	    - 聚合查询语句示例
+	- 后续工作
+		
+<!-- /TOC -->
+# TsFile的Hive连接器使用手册
+
+## 什么是TsFile的Hive连接器
+
+TsFile的Hive连接器实现了对Hive读取外部Tsfile类型的文件格式的支持，
+使用户能够通过Hive操作Tsfile。
+
+有了这个连接器，用户可以
+* 将单个Tsfile文件加载进Hive，不论文件是存储在本地文件系统或者是HDFS中
+* 将某个特定目录下的所有文件加载进Hive，不论文件是存储在本地文件系统或者是HDFS中
+* 使用HQL查询tsfile
+* 到现在为止, 写操作在hive-connector中还没有被支持. 所以, HQL中的insert操作是不被允许的
+
+## 系统环境要求
+
+|Hadoop Version |Hive Version | Java Version | TsFile |
+|-------------  |------------ | ------------ |------------ |
+| `2.7.3` or `3.2.1`       |    `2.3.6` or `3.1.2`  | `1.8`        | `0.8.0-SNAPSHOT`|
+
+> 注意：关于如何下载和使用Tsfile, 请参考以下链接: <https://github.com/apache/incubator-iotdb/tree/master/tsfile>。
+
+## 数据类型对应关系
+
+| TsFile 数据类型   | Hive 数据类型 |
+| ---------------- | --------------- |
+| BOOLEAN          | Boolean         |
+| INT32            | INT             |
+| INT64       	   | BIGINT          |
+| FLOAT       	   | Float           |
+| DOUBLE      	   | Double          |
+| TEXT      	   | STRING          |
+
+
+## 为Hive添加依赖jar包
+
+为了在Hive中使用Tsfile的hive连接器，我们需要把hive连接器的jar导入进hive。
+
+从 <https://github.com/apache/incubator-iotdb>下载完iotdb后, 你可以使用 `mvn clean package -pl hive-connector -am -Dmaven.test.skip=true`命令得到一个 `hive-connector-X.X.X-SNAPSHOT-jar-with-dependencies.jar`。
+
+然后在hive的命令行中，使用`add jar XXX`命令添加依赖。例如:
+
+```
+hive> add jar /Users/hive/incubator-iotdb/hive-connector/target/hive-connector-0.9.0-SNAPSHOT-jar-with-dependencies.jar;
+
+Added [/Users/hive/incubator-iotdb/hive-connector/target/hive-connector-0.9.0-SNAPSHOT-jar-with-dependencies.jar] to class path
+Added resources: [/Users/hive/incubator-iotdb/hive-connector/target/hive-connector-0.9.0-SNAPSHOT-jar-with-dependencies.jar]
+```
+
+
+## 创建Tsfile-backed的Hive表
+
+为了创建一个Tsfile-backed的表，需要将`serde`指定为`org.apache.iotdb.hive.TsFileSerDe`，
+将`inputformat`指定为`org.apache.iotdb.hive.TSFHiveInputFormat`，
+将`outputformat`指定为`org.apache.iotdb.hive.TSFHiveOutputFormat`。
+
+同时要提供一个只包含两个字段的Schema，这两个字段分别是`time_stamp`和`sensor_id`。
+`time_stamp`代表的是时间序列的时间值，`sensor_id`是你想要从tsfile文件中提取出来分析的传感器名称，比如说`sensor_1`。
+表的名字可以是hive所支持的任何表名。
+
+需要提供一个路径供hive-connector从其中拉取最新的数据。
+
+这个路径必须是一个指定的文件夹，这个文件夹可以在你的本地文件系统上，也可以在HDFS上，如果你启动了Hadoop的话。
```

Comment:
Better tell users what a directory path on the local file system should look like, e.g., `file:///home/username/dataDir/`.